src/Model/Notify.php

```diff
@@ -0,0 +1,775 @@
+		$diasp_url = str_replace('/profile/', '/u/', $myurl);
+
+		$condition = ["NOT `wall` AND `uid` = ? AND (`item`.`author-id` = ? OR `item`.`tag` REGEXP ? OR `item`.`tag` REGEXP ?)",
+			local_user(), public_contact(), $myurl . '\\]', $diasp_url . '\\]'];
+
+		if (!$seen) {
+			$condition[0] .= " AND `unseen`";
+		}
+
+		$fields = ['id', 'parent', 'verb', 'author-name', 'unseen', 'author-link', 'author-avatar', 'contact-avatar',
+			'network', 'created', 'object', 'parent-author-name', 'parent-author-link', 'parent-guid'];
+		$params = ['order' => ['received' => true], 'limit' => [$start, $limit]];
+
+		$items = Item::selectForUser(local_user(), $fields, $condition, $params);
+
+		if ($this->dba->isResult($items)) {
+			$notifies = $this->formatNotifies(Item::inArray($items), $ident);
+		}
+
+		$arr = [
+			'notifications' => $notifies,
+			'ident'         => $ident,
+		];
+
+		return $arr;
+	}
+
+	/**
+	 * @brief Get home notifications
+	 *
+	 * @param bool $seen    False => only include notifications into the query
+	 *                            which aren't marked as "seen"
+	 * @param int        $start   Start the query at this point
+	 * @param int        $limit   Maximum number of query results
+	 *
+	 * @return array [string, array]
+	 *    string 'ident' => Notification identifier
+	 *    array 'notifications' => Home notifications
+	 *
+	 * @throws Exception
+	 */
+	public function getHomeNotifies(bool $seen = false, int $start = 0, int $limit = self::DEFAULT_PAGE_LIMIT)
+	{
+		$ident    = self::HOME;
+		$notifies = [];
+
+		$condition = ['wall' => false, 'uid' => local_user()];
+
+		if (!$seen) {
+			$condition['unseen'] = true;
+		}
+
+		$fields = ['id', 'parent', 'verb', 'author-name', 'unseen', 'author-link', 'author-avatar', 'contact-avatar',
+			'network', 'created', 'object', 'parent-author-name', 'parent-author-link', 'parent-guid'];
+		$params = ['order' => ['received' => true], 'limit' => [$start, $limit]];
+
+		$items = Item::selectForUser(local_user(), $fields, $condition, $params);
+
+		if ($this->dba->isResult($items)) {
+			$notifies = $this->formatNotifies(Item::inArray($items), $ident);
+		}
+
+		$arr = [
+			'notifications' => $notifies,
+			'ident'         => $ident,
+		];
+
+		return $arr;
+	}
+
+	/**
+	 * @brief Get introductions
+	 *
+	 * @param bool $all     If false only include introductions into the query
+	 *                      which aren't marked as ignored
+	 * @param int  $start   Start the query at this point
+	 * @param int  $limit   Maximum number of query results
+	 * @param int  $id      When set, only the introduction with this id is displayed
+	 *
+	 * @return array [string, array]
+	 *    string 'ident' => Notification identifier
+	 *    array 'notifications' => Introductions
+	 *
+	 * @throws ImagickException
+	 * @throws Exception
+	 */
+	public function getIntroNotifies(bool $all = false, int $start = 0, int $limit = self::DEFAULT_PAGE_LIMIT, int $id = 0)
```

Comment:
`Notify` already is the name of the class, this should be renamed `getIntroList`.